bin/get_qa.cmd

```diff
@@ -0,0 +1,81 @@
+@ECHO OFF
+setlocal enabledelayedexpansion
+
+set branch=%1
+set build=%2
+set cpu=%3
+
+REM Check if parameters are set
+if %branch%.==. GOTO args_error
+if %build%.==. GOTO args_error
+if %cpu%.==. (
+	GOTO args_error
+) else (
+	GOTO set_env
+)
+
+:args_error
+echo User error: must specify branch, build type, CPU arch and revision code
+echo get_release "<branch> <build> <cpu>"
+echo Branch can be any of: 5.6.XRCX, 7.0.XRCX, 7.1.XRCX, 7.2.XRCX, 7.3.XRCX, 7.4.XRCX
```

Comment:
I don't think it makes sense to support 5.6 - 7.1 here, since there are no more QA releases for these branches. If there will be QA releases for the sec-backports, these will likely be available somewhere else (maybe even on another server).

Reply:
Updated and removed support for 5.6 - 7.1 (kept 7.1 for releases and snaps).